specification/azsadmin/resource-manager/backup/Microsoft.Backup.Admin/preview/2018-09-01/Backups.json

```diff
@@ -277,6 +285,19 @@
                     "type": "string"
                 }
             }
+        },
+        "RestoreOptions": {
+            "description": "Properties for restore options.",
+            "properties": {
+                "decryptionCertBase64": {
```

Comment:
So this is the private key, which means during backup we use the public key?

Reply:
Yes, public key to wrap the our internally generated encryption key during backup, and private key to decrypt that encryption key during restore.